src/connection/index.js

```diff
@@ -54,14 +54,14 @@ const connectionDefinitions = (
       # A cursor for use in pagination.
       cursor: String!
       # The item at the end of the edge
-      node: ${connectionNode}
+      node: ${connectionNode}!
       ${edgeFields ? edgeFields : ""}
     }
   
     # A connection to a list of items.
     type ${name}Connection {
       # A list of edges.
-      edges: [${name}Edge]
+      edges: [${name}Edge!]!
```

Comment:
And all the APIs always include the list of edges too

Reply:
In this project I followed the [relay connection specification](https://facebook.github.io/relay/graphql/connections.htm) and there the edges are nullable. I want to keep this implementation in line with the spec.